sample.tf

```diff
@@ -9,56 +9,161 @@ terraform {
 }
 
 provider "xray" {
-  //  supply ARTIFACTORY_USERNAME, _PASSWORD and _URL as env vars
+  //  supply ARTIFACTORY_URL and ARTIFACTORY_ACCESS_TOKEN as env vars
```

Comment:
Remove the word artifactory

Reply:
hm, they are technically Artifactory URL and Artifactory access token. There is no separate Xray URL. Token, yes, we can create a separate one for Xray only

Reply:
I don't know how people have setup thier platforms, but we definitely want the option to supply different ones or the same to all. 

So, we want: `XRAY_ACCESS_TOKEN` and `JFROG_ACCESS_TOKEN` - both artifactory and xray will share this last one. They are redundant by design